app/views/keywords/index.html.slim

```diff
@@ -0,0 +1,15 @@
+div class="d-flex flex-column align-items-center"
+
+  = render(partial: 'upload_csv', locals: { csv_import_form: csv_import_form })
+
+  - if keywords.present?
+    table class="table table-hover my-5 w-50"
+      thead.thead-light
+        tr
+          th = t('keyword.keyword')
+          th = t('keyword.created_at')
+      tbody
+        - keywords.each do |k|
```

Comment:
A small thing, nothing wrong with this implement, but I recommend you can have a look on https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections, because we use that style on the real project :)

Reply:
Fixed in da708b68749f714aa9fa88d4093bae72cb8b66ed